benchmarks/viscoelastic_plastic_shear_bands/gerya_2019/gerya_2019_vep.prm

```diff
@@ -0,0 +1,75 @@
+# This parameter file modifies the benchmark gerya_2019_vp.prm 
+# to include viscoelasticity.
+
+include $ASPECT_SOURCE_DIR/benchmarks/viscoelastic_plastic_shear_bands/gerya_2019/gerya_2019_vp.prm
+
+#  Global parameters
+set Dimension                              = 2
+set End time                               = 500
+set Output directory                       = output_gerya_2019_vep
+
+subsection Formulation
+  set Enable elasticity = true
+end
+
+# Number and name of compositional fields
+subsection Compositional fields
+  set Number of fields = 6
+  set Names of fields  = stress_xx, stress_yy, stress_xy, block, air, inclusion
+end
+
+# Spatial domain of different compositional fields
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  =
+    set Function expression = 0; 0; 0; \
+                              if ( ( x<43.75e3 && y>25.e3 && y<75.e3) || (x>56.25e3 && y>25.e3 && y<75.e3) || (y>56.25e3 && y<75.e3 && x>=43.75e3 && x<=56.25e3) || (y<43.75e3 && y>25.e3 && x>=43.75e3 && x<=56.25e3), 1, 0); \
+                              if (y<=25.e3 || y>=75.e3, 1, 0); \
+                              if (y<=56.25e3 && y>=43.75e3 && x>=43.75e3 && x<=56.25e3, 1, 0);  
+  end
+end
+
+# Material model
+subsection Material model
+
+  set Material averaging = harmonic average only viscosity
+
+  set Model name = visco plastic
+
+  subsection Visco Plastic
+
+    set Densities                   = 2700
+    set Reference viscosity         = 1.e21
+    set Reference strain rate       = 1.e-13
+    set Maximum viscosity           = 1.e23
+    set Minimum viscosity           = 1.e17
```

Comment:
The five parameters above are the same as in the original file, so you don't need them here. 
(unless you think it's important to list them as a reminder for the user)

Reply:
Good catch, no reason to keep these in.